sys/netbsd/dev_i2c.txt

```diff
@@ -0,0 +1,18 @@
+#Autogenerated by sys2syz
+include <i2c_io.h>
+
+resource fd_i2c[fd]
+
+syz_open_dev$I2C(dev ptr[in, string["/dev/i2c"]], id intptr, flags flags[open_flags]) fd_i2c
+
+ioctl$I2C_IOCTL_EXEC(fd fd_i2c, cmd const[I2C_IOCTL_EXEC], arg ptr[out, i2c_ioctl_exec])
+
+i2c_ioctl_exec {
+iie_op	flags[i2c_op_t_flags]
```

Comment:
When flags is used inside of a struct, it needs base type, e.g.: `flags[i2c_op_t_flags, int16]` otherwise there is no way to inject layout of this struct. It's like specifying a C struct field with name foo, but not saying how many bytes it occupies.

Reply:
> I can see errors like `wrong number of arguments for type flags, expect flags, base type`, can you have a look?

Still working on automating the fetching of macros used by flag type.

Reply:
> When flags is used inside of a struct, it needs base type, e.g.: `flags[i2c_op_t_flags, int16]` otherwise there is no way to infer layout of this struct. It's like specifying a C struct field with name foo, but not saying how many bytes it occupies.

Would surely take a note of it.